src/components/ImageView/index.js

```diff
@@ -57,12 +61,79 @@ class ImageView extends PureComponent {
         document.removeEventListener('mousemove', this.trackMovement.bind(this));
     }
 
+    onContainerLayoutChanged(e) {
```

Comment:
@ahmdshrif sorry, but requesting a really minor change.

Could you please add JSdoc for all the functions, thanks!